draft-ietf-taps-arch.md

```diff
@@ -123,7 +123,7 @@ This subsection provides a glossary of key terms related to the Transport Servic
 - Connection Group: A set of Connections that share properties and caches.
 - Connection Property: A Transport Property that controls per-Connection behavior of a Transport Services implementation.
 - Endpoint: An identifier for one side of a Connection (local or remote), such as a hostname or URL.
-- Equivalent Protocol Stacks: Protocol Stacks that can be safely swapped or raced in parallel during establishment of a Connection.
+- Equivalent Protocol Stack: A Protocol Stack that can be safely swapped or raced in parallel during establishment of a Connection.
```

Comment:
I'm against this part of the change. Equivalence is only defined for > 1 entities - you can't say "this protocol stack is equivalent" without saying "equivalent to what?".

So, I suggest to undo, here:
```suggestion
- Equivalent Protocol Stacks: Protocol Stacks that can be safely swapped or raced in parallel during establishment of a Connection.
```